src/shell/plugins/codec.c

```diff
@@ -90,6 +90,17 @@ int codec_data_decode (json_t *o, void **datap, size_t *lengthp)
     return 0;
 }
 
+ssize_t codec_data_length (json_t *o)
+{
+    void *data;
+    size_t length;
+    if (codec_data_decode (o, &data, &length) < 0)
+        return -1;
+    free (data);
```

Comment:
There's no way to get the length without a malloc/free :slightly_frowning_face: ?